packages/connect-popup/e2e/support/helpers.ts

```diff
@@ -20,3 +21,50 @@ export const findElementByDataTest = async (page: Page, dataTest: string, timeou
 export const log = (...val: string[]) => {
     console.log(`[===]`, ...val);
 };
+
+export const getExtensionPage = async () => {
```

Comment:
just an idea, not sure if feasible. But it would be nice to have something like `getExplorerPage` and `getPopupPage` functions that would be implemented diffferently for web and webextension and that would hide implementation details so that the person writing tests doesn't need to care

Reply:
Yes, that is a good idea, I did some sort of abstraction https://github.com/trezor/trezor-suite/pull/9578/commits/175ae327b674489f08b302cacb50e18d11b4e9cf

I think this helps to write tests without worrying to cover both environment, nonetheless is not bullet-proof since the person writing the tests might still have to consider it in some situations since events triggered behave diferent.